src/alphaInstaller.ts

```diff
@@ -0,0 +1,66 @@
+import * as semver from 'semver';
+import {window, commands, Uri,} from "vscode";
+import { Capability, isCapabilityEnabled } from "./capabilities";
+import { installCommand, latestReleaseUrl, minimalSupportedVscodeAPI } from "./consts";
+import { downloadFileToDestination, downloadFileToStr } from './download.utils';
+import { tabnineContext } from "./extensionContext";
+import createTempFileWithPostfix from './file.utils';
+
+type GitHubAsset = {
+  browser_download_url: string
+}
+type GitHubReleaseResponse = {
+  assets: GitHubAsset[],
+}
+export default async function handleAlpha(): Promise<void> {
+  try {
+
+    if (shouldCheckAvailableVersion(tabnineContext.vscodeVersion)) {
```

Comment:
no need to pass the import here, it adds no readability. Plus you can polish the name a bit so it will read like natural language in the if.. e.g.: `if (userConsumesAlphaVersions()) {`